### ISSUE TITLE
Cannot find module 'debug-utils' when trying to scaffold app

### ISSUE THREAD

**Issue by christianhuening** [ng-tools/generator-ng-factory#7]
Hi,

did 'npm i -g yo bower generator-ng-factory'
and then tried to 'yo ng-factory'
However I receive an error as follows:
```javascript
module.js:338
    throw err;
          ^
Error: Cannot find module 'debug-utils'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-ng-factory/app/index.js:8:1)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.defineProperty.get [as ng-factory:app] (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/store.js:40:23)
    at Store.get (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/store.js:64:35)
    at Environment.get (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/environment.js:243:16)
    at Environment.create (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/environment.js:261:24)
    at Environment.run (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/environment.js:333:24)
```

Of course I tried to google it, but no luck.

**Comment by zerojuan** [comment]
+1

**Comment by Hooter** [comment]
+1

**Comment by gcoupelant** [comment]
+1

**Comment by JoeRobles** [comment]
+1

**Comment by mgcrea** [comment]
Fixed by 9ea2cade4fae497645478574f524b746768637b4.
